core/src/test/java/org/elasticsearch/action/support/IndicesOptionsTests.java

```diff
@@ -60,29 +60,82 @@ public void testSerialization() throws Exception {
     }
 
     public void testFromOptions() {
-        int iterations = randomIntBetween(5, 20);
-        for (int i = 0; i < iterations; i++) {
```

Comment:
was this loop here removed intentionally? Now it will take several test runs to test all combinations.